lisp/ess-r-xref.el

```diff
@@ -28,9 +28,8 @@
 
 ;;; Code:
 
-(when (>= emacs-major-version 25)
-  (require 'subr-x)
-  (require 'xref))
+(require 'subr-x)
```

Comment:
This file could be loaded on emacs 24 as well. Also ELPA compiles it. Maybe `(require xref t)`?

Reply:
I thought it wouldn't be loaded because requires are guarded. MELPA doesn't use our Makefile?

Reply:
I think they just compile all el files within the directory.

Reply:
@lionel- what's the state of this? I would suggest we leave this part as it is. It's a good marker for the time when we finally drop emacs 24.